ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -132,11 +137,14 @@ public void getOutline(Outline outline) {
       super.getOutline(outline);
       return;
     }
-    if(!CSSConstants.isUndefined(mBorderRadius) && mBorderRadius > 0) {
-      float extraRadiusFromBorderWidth = (mBorderWidth != null)
-              ? mBorderWidth.get(Spacing.ALL) / 2f
-              : 0;
-      outline.setRoundRect(getBounds(), mBorderRadius + extraRadiusFromBorderWidth);
+    if((!CSSConstants.isUndefined(mBorderRadius) && mBorderRadius > 0) || mBorderCornerRadii != null) {
+      if (mPathForBorderRadiusOutline == null) {
```

Comment:
Do we need this check here? I think that what we should do instead is that we should set `mNeedUpdatePathForBorderRadius = true` in both `setRadius` methods to be consistent with the way how we update that property.